.github/workflows/publish-release-artifact.yml

```diff
@@ -0,0 +1,67 @@
+name: Publish artifact
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Set version
+        id: set-version
+        run: |
+          version=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
+          echo "::set-output name=project-version::$version"
+          echo "::set-env name=PROJECT_VERSION::$version"
+      - run: echo $PROJECT_VERSION
+      - name: Fetch artifact
+        id: fetch-artifact
+        run: |
+          GROUP_ID=io/jenkins/jenkinsfile-runner
+          ARTIFACT_ID=jenkinsfile-runner
+          FILE_NAME=jenkinsfile-runner
+          PROJECT_VERSION=${{ steps.set-version.outputs.project-version }}
+          echo "::set-output name=file-name::$FILE_NAME"
+          wget -q https://repo.jenkins-ci.org/releases/$GROUP_ID/$ARTIFACT_ID/$PROJECT_VERSION/$ARTIFACT_ID-$PROJECT_VERSION-sources.jar \
+            -O $FILE_NAME-$PROJECT_VERSION-sources.jar
+          wget -q https://repo.jenkins-ci.org/releases/$GROUP_ID/$ARTIFACT_ID/$PROJECT_VERSION/$ARTIFACT_ID-$PROJECT_VERSION.jar \
```

Comment:
It is not a vanilla package artifact. https://github.com/jenkinsci/jenkinsfile-runner/blob/master/app/pom.xml is a package with a minimum required plugin set. Vanilla package is here: https://github.com/jenkinsci/jenkinsfile-runner/blob/master/vanilla-package/pom.xml .

Reply:
yes it as was downloading jenkinsfile-runner-<version>-sources.jar and removed now.